Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -62,13 +58,37 @@ extension TodosViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
         if editingStyle == UITableViewCellEditingStyle.delete {
             todos[indexPath.section].remove(at: indexPath.row)
-            tableView.reloadData()
+            tableView.performBatchUpdates({ tableView.deleteRows(at: [indexPath], with: .automatic) }, completion: nil)
         }
     }
 }
 
+extension TodosViewController: TodoTableViewCellDelegate {
+    func tableViewCell(didSelectTitleAt indexPath: IndexPath) {
+        navigationController?.pushViewController(TodoFormViewController(todo: todos[indexPath.section][indexPath.row]), animated: true)
+    }
+
+    func tableViewCell(didSelectCheckBoxAt indexPath: IndexPath) {
+        // TODO: Save changes on the server
+        let finishIndexPath = IndexPath(row: indexPath.section * todos[1 - indexPath.section].count, section: 1 - indexPath.section)
+        let removedTodo = todos[indexPath.section].remove(at: indexPath.row)
+        removedTodo.completed = !removedTodo.completed
+        todos[finishIndexPath.section].insert(removedTodo, at: finishIndexPath.row)
+        todosView.tableView.performBatchUpdates({
```

Comment:
Apply code formatting comments from the previous PRs and move to `didSelectRow..`.